sdks/java/io/parquet/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOTest.java

```diff
@@ -89,14 +90,23 @@ public void testWriteAndRead() {
 
   @Test
   public void testWriteAndReadFiles() {
+    writeReadAssert(CompressionCodecName.UNCOMPRESSED);
```

Comment:
Remove this test, it is useless if you set the default I proposed above.

Reply:
ok